src/test/suite/scanner.test.ts

```diff
@@ -33,6 +34,8 @@ suite('ScanRunner', () => {
 
             // ===== TEST =====
             // Use the scan runner on our target list to create and return our arg array.
+            
+            /* eslint-disable-next-line @typescript-eslint/no-explicit-any */ // TODO: Wow - using "any" here to somehow get access to a private method. Why is this test written this way?
```

Comment:
This is a technique we used for certain tests in the V4 Era, and reused here. It's really helpful when you need your tests to be able to verify something about a private method or property or the like. And while we can look back and say that it's a problem because it tied the tests too closely to the implementation detail, it did seem like a good idea at the time.